### PR TITLE
Creating locale string for openWorkflow.filterWorkflows & use it in the Open Workflow Modal

### PR DESCRIPTION
**Pull request by wohfab**
Combining #2714 & #2715 in one PR.

### CODE REVIEW

File: packages/editor-ui/src/components/WorkflowOpen.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			<template v-slot:header>
 				<div class="workflows-header">
 					<n8n-heading tag="h1" size="xlarge" class="title">
-						{{ $locale.baseText('workflowOpen.openWorkflow') }}
+						{{ $locale.baseText('workflowOpen.filterWorkflows') }}
```
Comment (mutdmour) [inline packages/editor-ui/src/components/WorkflowOpen.vue:11]: this updates the title of the modal. Did you mean to change the placeholder text instead in Tags Dropdown component below on line 15?

Comment (wohfab) [reply packages/editor-ui/src/components/WorkflowOpen.vue:11]: Oh man, I was not very concentrated there. Yes, line 15 was the intended target for my change.

Comment (mutdmour) [reply packages/editor-ui/src/components/WorkflowOpen.vue:11]: you need to update the title below

Comment (mutdmour) [reply packages/editor-ui/src/components/WorkflowOpen.vue:11]: yes same branch please

Comment (wohfab) [reply packages/editor-ui/src/components/WorkflowOpen.vue]: Alright! Done. Thank you very much for the guidance.

**Comment by janober** [comment]
Great, thanks a lot for fixing this! Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.160.0`

**Pull request merged**